tests/tests/cpld_control_c96_noaero_p8

```diff
@@ -0,0 +1,105 @@
+#
+#  cpld_control C96 noaero P8 test
```

Comment:
Should we call this "nowave_noareo_p8", or perhaps "s2s_p8" (where the std config is s2swa).